packages/belos/tpetra/src/solvers/Belos_Tpetra_GmresSstep.hpp

```diff
@@ -364,15 +356,43 @@ class GmresSstep : public Gmres<SC, MV, OP>  {
               T(i, iter+iiter) = H(i, iter+iiter);
             }
             this->reduceHessenburgToTriangular(iter+iiter, T, cs, sn, y);
+            if (outPtr != nullptr) {
+              *outPtr << " > implicit residual norm=(" << iter+iiter+1 << ")=" << STS::magnitude (y(iter+iiter+1))
```

Comment:
Prefer lines no longer than 70 characters.  This can make reading easier for the visually impaired.